Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -322,6 +348,35 @@
     completionHandler();
 }
 
+void WebPage::enableTextIndicatorStyleAfterElementWithID(const String& elementID, const WTF::UUID& uuid)
+{
+    // FIXME: move the start of the range to be after the element with the ID listed.
+
+    RefPtr frame = m_page->checkedFocusController()->focusedOrMainFrame();
+    if (!frame) {
+        ASSERT_NOT_REACHED();
+        return;
+    }
+    
+    RefPtr document = frame->document();
+    if (!document) {
+        ASSERT_NOT_REACHED();
+        return;
+    }
+    
+    RefPtr root = document->documentElement();
+    VisibleSelection fullDocumentSelection(VisibleSelection::selectionFromContentsOfNode(root.get()));
+    auto simpleRange = fullDocumentSelection.range();
+    if (!simpleRange) {
+        ASSERT_NOT_REACHED();
+        return;
+    }
+    
+    auto liveRange = createLiveRange(*simpleRange);
+
+    m_textIndicatorStyleEnablementRanges.set(uuid, liveRange);
```

Comment:
Nit - `add` is a tiny bit more efficient in the case where you don't expect to be replacing any existing entries. You can also use `WTFMove` here to prevent ref-churn (or simply remove the temporary variable)

```suggestion
    m_textIndicatorStyleEnablementRanges.add(uuid, createLiveRange(*simpleRange));
```